rehive/api/resources/admin_resources.py

```diff
@@ -618,3 +621,12 @@ def register(self,
         }
         response = self.post(data, 'register', **kwargs)
         return response
+
+
+class APIAdminCompanyAddress(ResourceList):
```

Comment:
Why is this a `ResourceList`. there can only be 1 resource returned on the `/admin/company/address/` endpoint. I think this should instead inherit from `Resource`.